scripts/build/arch/arm.sh

```diff
@@ -39,3 +39,16 @@ CT_DoArchTupleValues() {
         CT_TARGET_SYS="${CT_TARGET_SYS}hf"
     fi
 }
+
+#------------------------------------------------------------------------------
+# Get multilib architecture-specific target
+# Usage: CT_DoArchMultilibTarget "multilib flags" "target tuple"
+CT_DoArchMultilibTarget ()
+{
+    local multi_flags="${1}"
+    local target="${2}"
+
+    :;
```

Comment:
Agreed, but this was the change in @mingwandroid's branch which I used as a starting point. I only fixed them where they were broken, didn't do any cosmetic changes at this point as I was pretty sure they'll end up heavily modified anyway. In one of the later commits, these functions are dropped on the architectures where they do nothing.